infrastructure/oss/code_actions.go

```diff
@@ -169,7 +169,11 @@ func (i *ossIssue) getQuickfixEdit(affectedFilePath string) string {
 }
 
 func (i *ossIssue) getUpgradedPathParts() (string, string) {
-	rootDependencyUpgrade := strings.Split(i.UpgradePath[1].(string), "@")
+	s, ok := i.UpgradePath[1].(string)
+	if !ok {
+		return "", ""
+	}
```

Comment:
This is used to construct code actions - and if the cast fails, it would just not add a code action, but not pop up an error to the client. I can make this explicit though, by updating the signature. I don't like to panicking without cause...